WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -424,7 +406,7 @@ extension InPersonPaymentsMenuViewController {
     }
 
     @objc func learnMoreAboutInPersonPaymentsButtonWasTapped() {
-        WebviewHelper.launch(WooConstants.URLs.inPersonPaymentsLearnMoreWCPay.asURL(), with: self)
+        WebviewHelper.launch(inPersonPaymentsLearnMoreViewModel.url, with: self)
```

Comment:
I see we're switching the WooConstant `https://docs.woocommerce.com/document/getting-started-with-in-person-payments-with-woocommerce-payments/` for the following static let in the view model:
```
static let learnMoreURL = URL(string: "woocommerce://in-person-payments/learn-more")!
```

Should me move the `woocommerce://in-person-payments/learn-more` string to WooConstants? I see as well that both redirect to the same page ( as expected ), but I can't seem to find how these `woocommerce://` deep links are  setup. I though these were added into the WooCommerce target > Info > URL Types, but it doesn't seem to be the case, could you point me to the right direction with these?

Reply:
> I see we're switching the WooConstant https://docs.woocommerce.com/document/getting-started-with-in-person-payments-with-woocommerce-payments/ for the following static let in the view model:

Actually, we are still using `WooConstants.URLs.inPersonPaymentsLearnMoreWCPay.asURL() `, the only difference is that now we are initializing that in the `func inPersonPayments() -> LearnMoreViewModel` function in `LearnMoreViewModel`.

To be honest, I don't know what's the exact point of `woocommerce://in-person-payments/learn-more`; as you mentioned that protocol is not setup for deep links. My guess is that we use it just as a template, so we can use it to compose other URLs. But that's just my assumption, I would have to dig deeper to fully understand it. As it's not related to this PR I wouldn't change it in any way, but we can open an issue to verify this -and other usages- of the `woocommerce` URL protocol .